libs/net/dal/Migrations/Initial/Up/PostUp/01-DataSource.sql

```diff
@@ -2916,20 +2916,44 @@ INSERT INTO public.data_source (
   , ''
 ), (
   'CBCV'
-  , 'CBCV'
+  , 'CBCV-CAPTURE'
   , ''
   , true -- is_enabled
-  , 1 -- content_type_id
+  , NULL -- content_type_id
   , 3 -- media_type_id
   , 2 -- data_location_id
   , 3 -- license_id
   , ''
-  , '{}' -- connection
+  , '{
+    "serviceType":"stream",
+    "url":"https://cbcrclinear-tor.akamaized.net/hls/live/2042769/geo_allow_ca/CBCRCLINEAR_TOR_15/master4.m3u8",
+    "timezone":"UTC",
```

Comment:
Having two of these doesn't appear to be valid.  I see the case difference, but we shouldn't need both.  If you saw two values somewhere it would probably mean we have a bug.